dm/master/server.go

```diff
@@ -2027,3 +2323,384 @@ func (s *Server) workerArgsExtractor(args ...interface{}) (workerrpc.Client, str
 
 	return cli, worker, nil
 }
+
+// WatchRequest watches requests in etcd, and handle these request, and write response to etcd.
+func (s *Server) WatchRequest(ctx context.Context) {
+	watchCh := s.etcdClient.Watch(ctx, defaultOperatePath, -1)
+
+	for {
+		select {
+		case <-ctx.Done():
+			return
```

Comment:
add  some log to indicate that the goroutine exited, here or in defer function

Reply:
add in b091a46